isaric/parsers/isaric-ccpuk.toml

```diff
@@ -1020,6 +1025,18 @@
   phase = "study"
   date = { field = "daily_dsstdat", description = "Date of assessment" }
 
+[[observation]]
+  name = "fio2_percent"
+  phase = "study"
+  date = { field = "daily_dsstdat", description = "Date of assessment" }
+  value = { field = "daily_fio2b_lborres", apply = { function = "Percentage" } }
```

Comment:
daily_fio2b_lborres is already provided as a percentage, so is the "Percentage" function not needed here?
daily_fio2_lborres is provided as a fraction of 1 so the percentage function looks ok for this one